From: Debian Python Modules Team
 <python-modules-team@lists.alioth.debian.org>
Date: Tue, 15 Jan 2019 00:39:20 -0500
Subject: no_vendored_html5lib

Updated for bleach 3.2.0 by Leo <thinkabit.ukim@gmail.com>

===================================================================
diff --git a/bleach/html5lib_shim.py b/bleach/html5lib_shim.py
index 62ad0a6..e924bcf 100644
--- a/bleach/html5lib_shim.py
+++ b/bleach/html5lib_shim.py
@@ -15,23 +15,23 @@ import six
 # filter out html5lib deprecation warnings to use bleach
 warnings.simplefilter("ignore", category=DeprecationWarning)
 
-from bleach._vendor.html5lib import (
+from html5lib import (
     HTMLParser,
     getTreeWalker,
 )
-from bleach._vendor.html5lib import constants
-from bleach._vendor.html5lib.constants import (
+from html5lib import constants
+from html5lib.constants import (
     namespaces,
     prefixes,
 )
-from bleach._vendor.html5lib.constants import _ReparseException as ReparseException
-from bleach._vendor.html5lib.filters.base import Filter
-from bleach._vendor.html5lib.filters.sanitizer import allowed_protocols
-from bleach._vendor.html5lib.filters.sanitizer import Filter as SanitizerFilter
-from bleach._vendor.html5lib._inputstream import HTMLInputStream
-from bleach._vendor.html5lib.serializer import HTMLSerializer
-from bleach._vendor.html5lib._tokenizer import attributeMap, HTMLTokenizer
-from bleach._vendor.html5lib._trie import Trie
+from html5lib.constants import _ReparseException as ReparseException
+from html5lib.filters.base import Filter
+from html5lib.filters.sanitizer import allowed_protocols
+from html5lib.filters.sanitizer import Filter as SanitizerFilter
+from html5lib._inputstream import HTMLInputStream
+from html5lib.serializer import HTMLSerializer
+from html5lib._tokenizer import attributeMap, HTMLTokenizer
+from html5lib._trie import Trie
 
 
 #: Map of entity name to expanded entity
diff --git a/tests/test_clean.py b/tests/test_clean.py
index 133cd82..2c0a1b9 100644
--- a/tests/test_clean.py
+++ b/tests/test_clean.py
@@ -7,7 +7,7 @@ import pytest
 from bleach import clean
 from bleach.html5lib_shim import Filter
 from bleach.sanitizer import Cleaner
-from bleach._vendor.html5lib.constants import rcdataElements
+from html5lib.constants import rcdataElements
 
 def test_clean_idempotent():
     """Make sure that applying the filter twice doesn't change anything."""
